### PR TITLE
feat: add ops dependencies

### PR DESCRIPTION
**Pull request by joshuarli**
See https://github.com/getsentry/ops/pull/5255.

### CODE REVIEW

File: packages.ini [hunk 1]
```diff
@@ -354,13 +380,16 @@ brew_requires = libmaxminddb
 
 [packaging==21.3]
 
+[paramiko==2.8.0]
```
Comment (asottile-sentry) [inline packages.ini:383]: looks like this needs to be at least 2.9.3 to be fixed: https://github.com/paramiko/paramiko/commit/11232cca7be7bc5fc51eef1e8869db4450bc79e3

File: packages.ini [hunk 2]
```diff
@@ -27,11 +27,15 @@
...
 
+[bcrypt==4.0.0]
+
 [beautifulsoup4==4.7.1]
 [beautifulsoup4==4.11.1]
 
 [billiard==3.6.4.0]
 
+[black==21.11b1]
+validate_extras = d
```
Comment (asottile-sentry) [inline packages.ini:38]: I would try upgrading ops -- this is a pretty old version of `black` with the annoying `click` bug

Comment (joshuarli) [reply packages.ini:38]: i vaguely remember there was a reason we were click<8.0, i'll revisit it

**Comment by joshuarli** [comment]
i'm gonna break this down into smaller PRs

**Pull request closed without merging**